code/game/objects/structures/ai_core.dm

```diff
@@ -72,7 +72,7 @@
 	. = ..()
 	circuit = new(src)
 	if(posibrain)
-		core_mmi = new/obj/item/mmi/posibrain(src)
+		core_mmi = new/obj/item/mmi/posibrain(src, FALSE)
```

Comment:
Named parameter. Because this is new you might have to do it like:

```suggestion
		core_mmi = new/obj/item/mmi/posibrain(src, /* whatever_its_called = */ FALSE)
```